tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py

```diff
@@ -903,7 +903,34 @@ def test_getnnz(self, xp, sp):
 
 
 @testing.with_requires('scipy>=0.19.0')
-class TestCsrMatrixScipyCompressedMinMax(unittest.TestCase):
+class TestCscMatrixScipyCompressedMinMax(unittest.TestCase):
+
+    def test_min_sparse_axis_none(self):
+        dm_data = numpy.random.random((10, 20))
+        dm_data[dm_data < 0.95] = 0
+
+        dm_data = scipy.sparse.csc_matrix(dm_data)
+        cp_matrix = sparse.csc_matrix((cupy.array(dm_data.data),
+                                       cupy.array(dm_data.indices),
+                                       cupy.array(dm_data.indptr)),
+                                      shape=(10, 20))
```

Comment:
I wonder if we could just call `cp_matrix = sparse.csc_matrix(dm_data)`? (I understand this matches the style in the other tests, just wondering).

Reply:
Yes, I think so.